HISTORY.rst

```diff
@@ -133,6 +133,7 @@ Unreleased Changes (3.9)
       value. Now, the decay does not ignore those nodata edges causing values
       on the edges to decay more quickly. The area of study should have
       adequate boundaries to account for these edge effects.
+    * Update default half saturation value for sample data to 0.05 from 0.5.
```

Comment:
It's trivial, but the former value was 0.1 https://bitbucket.org/natcap/invest-sample-data/commits/b7a51f189315e08484b5ba997a5c1de88ab7f06d